site/_data/authorsData.json

```diff
@@ -779,5 +779,23 @@
     },
     "country": "US",
     "image": "image/SHhb2PDKzXTggPGAYpv8JgR81pX2/kUhkXfBItc8GVTkWA2AB.jpg"
+  },
+  "hanselfmu": {
```

Comment:
Would you mind removing this. Instead I think we would like to add a line of italicized text at the bottom of the article that says: "Translation provided by [@hanselfmu](https://github.com/hanselfmu)."

We think it will be easier to scale this approach if we end up having a lot of community translations in the future.

Reply:
Sure, this is fine by me. Do you also want me to add this new line in this PR? Or would this line at the bottom belong to the parent scope of this article, thus won't go into this PR?

Reply:
> Do you also want me to add this new line in this PR?

Yes please.

> Or would this line at the bottom belong to the parent scope of this article, thus won't go into this PR?

I think for now we will only have it in this article (not the parent).

Reply:
Done. PTAL again and adjust to your requirement. Thanks.